elliottlib/attach_cve_flaws.py

```diff
@@ -36,25 +33,20 @@ def get_corresponding_flaw_bugs(bzapi, tracker_bugs):
     return [flaw_bug for flaw_bug in blocking_bugs if is_flaw_bug(flaw_bug)]
 
 
-def is_first_fix(bzapi, flaw_bug, current_target_release, tracker_ids_to_be_ignored=[]):
+def is_first_fix(bzapi, flaw_bug, tracker_ids_to_be_ignored=[]):
     other_flaw_trackers = bzapi.query(bzapi.build_query(
         product='OpenShift Container Platform',
         bug_id=[t for t in flaw_bug.depends_on if t not in tracker_ids_to_be_ignored],
     ))
 
-    def _filter_tracker(bug):
```

Comment:
This only-major-version-match filter is removed in favor of a strict check of 1 exact target release value for all bugs